pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-7248.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet author="Dmitrii Eliseev" id="20220427-audit-table">
+        <createTable tableName="audit_trail">
+            <column name="audit_trail_id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" primaryKeyName="audit_trail_pk"/>
+            </column>
+
+            <column name="study_id" type="bigint" />
+
+            <column name="operator_id" type="bigint">
+                <constraints nullable="true" />
```

Comment:
Isn't nullable=true by default any way ?
Same applies to all below columns too

Reply:
Changed